### ISSUE TITLE
Possible to add more fields Mega Menu options?

### ISSUE THREAD

**Issue by bowenac** [familiar-studio/craft-menus#7]
I was thinking it would be nice to have a couple more fields in addition to custom url.
An image field.
Summary field.
And as someone else mentioned css class.

This menu plugin is great and works great but I need to create a mega menu of sorts that will show the title, a short summary, and a button which would be the link, no images but that would just be a nice feature to have for mega menus.

Then I could just check if the nodes are child nodes to determine if those fields are used etc as it would then be considered part of a mega menu.

**Comment by keithmancuso** [comment]
Agreed i had originally resisted this but it seems like it would be nice to be able to add custom fields to menu nodes, working on an update now to address
